frontend/src/metabase-lib/v1/expressions/errors.ts

```diff
@@ -34,27 +34,6 @@ export class CompileError extends ExpressionError {
   }
 }
 
-export class ResolverError extends ExpressionError {
```

Comment:
This is now just a `CompilerError`, we don't really use the error types for anything anyway.